sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAsyncClient.java

```diff
@@ -58,7 +60,8 @@
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<AttestationResponse> attestOpenEnclave(AttestOpenEnclaveRequest request) {
-        return this.serviceClient.attestOpenEnclaveAsync(request);
+        return serviceClient.attestOpenEnclaveAsync(request.toGenerated())
+            .map(result -> AttestationResponse.fromGenerated(result));
```

Comment:
you could change this to .map(AttestationResponse::fromGenerated)

Reply:
Great suggestion, implemented.